src/resonanceReconstruction/rmatrix/legacy/resolved/SpinGroup/SingleLevelBreitWigner/src/precompute.hpp

```diff
@@ -0,0 +1,30 @@
+void precompute( const Energy& energy ) {
```

Comment:
It's unlike @whaeck to not have documentation for each function. Are you sure you wrote this? 😉

Reply:
That's because that function is an internal only one. I never document a function that's internal only.